src/python/pants/backend/python/targets/python_requirement_library.py

```diff
@@ -17,19 +17,43 @@ class PythonRequirementLibrary(Target):
   :API: public
   """
 
-  def __init__(self, payload=None, requirements=None, **kwargs):
+  def __init__(self, payload=None, requirements=None, prepend_to_pythonpath=False, **kwargs):
     """
+    NB: google-cloud-bigquery==1.10.0 has the following prelude upon import:
```

Comment:
I'd generify this. "Certain libraries, such as google-cloud-bigquery==1.10.0, use the following prelude upon import"